### PR TITLE
frontend header behaviour

### PR DESCRIPTION
**Pull request by akssinhaamit**
# What and why?
Take out header Help from all the pages except login page and help page
# How to test?
smoke test
# Trello
https://trello.com/c/WqKcyv2O/565-user-self-serve-frontstage-header-behaviour

### CODE REVIEW

File: frontstage/templates/layouts/configs/service-links.html [hunk 1]
```diff
@@ -3,21 +3,33 @@
...
-    %}
+    {% if ns and ns.showHelp %}
+        {% do serviceLinks | setAttribute("itemsList", [
+            {
+              "text": "My Account",
+              "url": "/my-account"
+            },
+            {
+              "text": "Help",
+              "url": ""
```
Comment (mfcrocker) [inline frontstage/templates/layouts/configs/service-links.html:14]: Is this right?

**Review by mfcrocker — APPROVED** [review]
One question to check up on - I can't test this properly because my database is pretty borked right now but looks good other than that.

File: frontstage/templates/layouts/configs/service-links.html (hunk 1 above)
Comment (akssinhaamit) [reply frontstage/templates/layouts/configs/service-links.html]: changed as per the review comment

**Review by mfcrocker — APPROVED** [review]
Happy with changes

**Pull request merged**